HandleNHNotificationCall/__tests__/handler.test.ts

```diff
@@ -75,28 +75,28 @@ describe("HandleNHNotificationCall", () => {
     await getHandler()(context as any, aNotifyMessage);
 
     expect(dfClient.startNew).toHaveBeenCalledWith(
-      "HandleNHNotificationCallOrchestrator",
+      "HandleNHNotifyMessageCallOrchestrator",
       undefined,
       {
         message: aNotifyMessage
       }
     );
   });
 
-  // it("should not call any Orchestrator when message kind is not correct", async () => {
-  //   const aWrongMessage = {
-  //     installationId: aFiscalCodeHash,
-  //     kind: "WrongMessage" as any
-  //   };
-
-  //   // tslint:disable-next-line: no-let
-  //   let hasError = false;
-  //   try {
-  //     await HandleNHNotificationCall(context as any, aWrongMessage);
-  //   } catch (error) {
-  //     hasError = true;
-  //   }
-
-  //   expect(hasError).toBe(true);
-  // });
+  it("should not call any Orchestrator when message kind is not correct", async () => {
+    const aWrongMessage = {
+      installationId: aFiscalCodeHash,
+      kind: "WrongMessage" as any
+    };
+
+    // tslint:disable-next-line: no-let
+    let hasError = false;
+    try {
+      await getHandler()(context as any, aWrongMessage);
+    } catch (error) {
+      hasError = true;
+    }
+
+    expect(hasError).toBe(true);
```

Comment:
what about 

```ts
const result = await getHandler()(context as any, aWrongMessage);
expect(result).toEqual(whatever);
// or
expect(result).toBe(undefined);
```
?

Reply:
Or (not tried)
```ts
expect.assertions(0);
await getHandler()(context as any, aWrongMessage);
```

Reply:
We expect it to throw an error, so we cannot get a result. 
So I want to be sure that this error has been thrown; I think I could have done something like this:
```suggestion
    // tslint:disable-next-line: no-let
    expect.assertions(1);
    try {
      await getHandler()(context as any, aWrongMessage);
    } catch (error) {
      expect(hasError).toBe(true);
    }
```